Content.Shared/Clothing/EntitySystems/EmitsSoundOnMoveSystem.cs

```diff
@@ -0,0 +1,96 @@
+using System.Numerics;
+using Content.Shared.Clothing.Components;
+using Content.Shared.Gravity;
+using Content.Shared.Inventory;
+using Content.Shared.Inventory.Events;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Movement.Components;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Physics.Components;
+using Robust.Shared.Timing;
+
+namespace Content.Shared.Clothing.Systems;
+
+public sealed class EmitsSoundOnMoveSystem : EntitySystem
```

Comment:
Refer to above, EmitsSoundOnMove needs to go into a separate PR

Reply:
This is needed for this PR, i am not making a new PR just for for this system.